bokeh/client/session.py

```diff
@@ -256,6 +261,7 @@ def close(self, why="closed"):
         self._connection.close(why)
 
     def loop_until_closed(self):
+        [cb.start() for cb in self._callbacks]
```

Comment:
I think you want to start callbacks immediately, there's no requirement that anyone ever calls `loop_until_closed`